2022-05-27 08:36:57,446 - CarbonIntensityThread - INFO - Traceback (most recent call last):
  File "/home/jwg356/nascar/vendors/nc_carbontracker/carbontracker/emissions/intensity/intensity.py", line 72, in carbon_intensity
    carbon_intensity = fetcher.carbon_intensity(g_location, time_dur)
  File "/home/jwg356/nascar/vendors/nc_carbontracker/carbontracker/emissions/intensity/fetchers/energidataservice.py", line 19, in carbon_intensity
    ci = self._emission_current()
  File "/home/jwg356/nascar/vendors/nc_carbontracker/carbontracker/emissions/intensity/fetchers/energidataservice.py", line 46, in _emission_current
    raise exceptions.CarbonIntensityFetcherError(response.json())
carbontracker.exceptions.CarbonIntensityFetcherError: {'statusCode': 429, 'message': 'Rate limit is exceeded. Try again in 10 seconds.'}

